src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/category/CategoryPageObject.java

```diff
@@ -30,4 +36,21 @@ public void verifyCategoryPageTitle(String categoryLinkName) {
         );
         Assertion.assertEquals("Category:" + title, categoryLinkName);
     }
+
+    public int getArticleIndexInGalleryByName(String articleName) {
+        int index = -1;
+        for(int i=0; i<categoryGalleryItemsName.size(); i++)
+        {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this left curly brace to the end of previous line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)